tests/features/entity-generator/EntityGenerator.test.ts

```diff
@@ -151,6 +151,23 @@ describe('EntityGenerator', () => {
     await orm.close(true);
   });
 
+  test('table name with underscore using entitySchema [mysql]', async () => {
+    const orm = await initORMMySql('mysql', {entityGenerator: {
```

Comment:
is this passing the linter? :] i dont think so, the indents on following lines are also wrong

```suggestion
    const orm = await initORMMySql('mysql', { entityGenerator: {
```

Reply:
true that. i didn't run lint from my local